primitives/benches/crypto_primitives/poseidon_crh.rs

```diff
@@ -8,20 +8,19 @@ use algebra::fields::mnt6753::Fr as MNT6753Fr;
 use algebra::UniformRand;
 use rand_xorshift::XorShiftRng;
 use rand::SeedableRng;
-use primitives::crh::poseidon::parameters::{MNT4753PoseidonParameters, MNT6753PoseidonParameters};
+use primitives::crh::poseidon::{
+    PoseidonParameters, parameters::{MNT4753PoseidonParameters, MNT6753PoseidonParameters}
+};
 use primitives::crh::poseidon::batched_crh::PoseidonBatchHash;
 
 fn poseidon_crh_eval_mnt4(c: &mut Criterion) {
 
     let mut rng = XorShiftRng::seed_from_u64(1231275789u64);
 
-    let mut input = Vec::new();
-    input.push(MNT4753Fr::rand(&mut rng));
-    input.push(MNT4753Fr::rand(&mut rng));
-
     c.bench_function("Poseidon CRH Eval for MNT4", move |b| {
         b.iter(|| {
-            MNT4PoseidonHash::evaluate(&input).unwrap();
+            let input = vec![MNT4753Fr::rand(&mut rng); 100];
+            MNT4PoseidonHash::evaluate(input.as_slice()).unwrap();
```

Comment:
Here, you're evaluating 50 hashes. Perhaps it is more intuitive to measure 100 or 1000 hashes.

Reply:
I tried to reflect the behaviour of the benchmark of normal Poseidon in order to be able to do comparison. Isn't it the same ? Aren't the benches for normal Poseidon also take 100 inputs, i.e. performing 50 permutations (hashes) ?